### PR TITLE
block retries

### PR DESCRIPTION
**Pull request by paulotelles**
Included verification of retries and two new envs to configure retries:

LOGIN_ATTEMPTS_TIME: configure the time in seconds for blocking the account to prevent too much retries.
LOGIN_ATTEMPTS_MAX_RETRIES: configure max retries before blocking the user.

### CODE REVIEW

File: lib/routes/api/login.js [hunk 1]
```diff
@@ -31,8 +31,28 @@ router.post('/', async(req, res) => {
       return res.sendStatus(403);
     }
     logger.info({r}, 'successfully retrieved user account');
+
+    const maxLogginAttempts = process.env.LOGIN_ATTEMPTS_MAX_RETRIES || 6;
```
Comment (davehorton) [inline lib/routes/api/login.js:35]: typo can you change var name to `maxLoginAttempts`

File: lib/routes/api/login.js [hunk 2]
```diff
@@ -31,8 +31,28 @@ router.post('/', async(req, res) => {
...
+    const logginAttempsBlocked = await retrieveKey(`login:${r[0].user_sid}`) >= maxLogginAttempts;
```
Comment (davehorton) [inline lib/routes/api/login.js:36]: typo, `loginAttemptsBlocked`

**Review by davehorton — CHANGES_REQUESTED** [review]

File: lib/routes/api/login.js (hunk 1 above)
Comment (paulotelles) [reply lib/routes/api/login.js]: fixed

File: lib/routes/api/login.js (hunk 2 above)
Comment (paulotelles) [reply lib/routes/api/login.js]: fixed

**Review by davehorton — APPROVED** [review]

**Pull request merged**